recipes/pytesseract/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pytesseract" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/madmaze/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: b389cf1208df8567f953b8da5b716b28f4c42f149f367b4e51b73ef651a55707
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Can you use the install line from the example, https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

```yaml
script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
@scopatz I was looking at [click](https://github.com/conda-forge/conda-smithy/blob/7ee8fb785642d2d5ce5244aa7db2d9912bbc45f0/tests/recipes/click-test-feedstock/recipe/meta.yaml#L19) and [pytest-mock](https://github.com/conda-forge/pytest-mock-feedstock/blob/c99e9c7add9dc7ae0d7445a1b097d235e14ee9ea/recipe/meta.yaml#L17)